deploy-hs

```diff
@@ -86,5 +86,9 @@ docker exec $CID python manage.py makemigrations
 echo "*** manage.py migrate ***"
 docker exec $CID python manage.py migrate
 
+docker exec $CID python manage.py migrate hs_core zero
+
+docker exec $CID python manage.py migrate
+
```

Comment:
This is not a file that I edited.  Shouldn't this be updated when I pull develop?  Am I doing something wrong?